resources/views/manager/profile/layout.html.twig

```diff
@@ -4,6 +4,13 @@
     Manager - Profile Layout
 
 ============================================================================= #}
+{% set managerEN = manager.translate('en') %}
```

Comment:
I'd like these to be set in the controller and passed in as well, but that's more of a personal preference thing